### DISCUSSION TITLE
Do I need to a repository for each project code?

### DISCUSSION THREAD

**Discussion by SelMesal** [community/community#43428] [New to GitHub]
Hi,

I am new to github.
If a team has three projects (three code sets), do we need three repositories on github?

**Comment by RONIE70** [comment]
GitHub está diseñado alrededor de un flujo de trabajo de colaboración específico, centrado en las solicitudes de integración (“pull request”). Este flujo es válido tanto si colaboras con un pequeño equipo en un repositorio compartido, como si lo haces en una gran red de participantes con docenas de bifurcaciones particulares. Se centra en el workflow [Ramas Puntuales](https://git-scm.com/book/es/v2/ch00/r_topic_branch) cubierto en [[ch03-git-branching]](https://git-scm.com/book/es/v2/ch00/ch03-git-branching).

El funcionamiento habitual es el siguiente:

Se crea una rama a partir de master.

Se realizan algunos commits hacia esa rama.

Se envía esa rama hacia tu copia (fork) del proyecto.

Abres un Pull Request en GitHub.

Se participa en la discusión asociada y, opcionalmente, se realizan nuevos commits.

El propietario del proyecto original cierra el Pull Request, bien fusionando la rama con tus cambios o bien rechazándolos.

Este es, básicamente, el flujo de trabajo del Responsable de Integración visto en [Flujo de Trabajo Administrador-Integración](https://git-scm.com/book/es/v2/ch00/r_integration_manager), pero en lugar de usar el correo para comunicarnos y revisar los cambios, lo que se hace es usar las herramientas web de GitHub.
https://git-scm.com/book/es/v2/GitHub-Participando-en-Proyectos

**Comment by ZososoUtsav** [comment]
Yes it's always better
